cdisutils/storage.py

```diff
@@ -46,7 +46,7 @@ class S3ConnectionProxyFix(S3Connection):
     - https://github.com/boto/boto/pull/3699
     """
     def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+        super(S3ConnectionProxyFix, self).__init__(*args, **kwargs)
```

Comment:
The docstring suggests this function only deals with Python3 code. If that's the case, would it be possible to use the more simplified super() syntax instead of super(subclass, instance)?

Reply:
I think everything that uses this class has probably been upgraded to Python 3 but the tox environment includes py27, so the old style `super()` is needed.